### PR TITLE
chore: mergeback for 0.9.0 changes

### PR DESCRIPTION
**Pull request by sfoster1**
This is the mergeback for changes from the 0.9.0 internal release process.

### CODE REVIEW

**Comment by sfoster1** [comment]
Conflicts are pretty simple - `helpers_ot3` needs to take the changes from `edge` to fix some lint issues

File: api/release-notes-internal.md [hunk 1]
```diff
@@ -12,10 +12,25 @@ Some things are known not to work, and are listed below. Specific compatibility
...
+- Motor driver configuration changes should improve performance and prevent step loss on the gantry.
+- More USB connectivity fixes; updating should now work
+- Many 96-channel behavior fixes, especially around tip pickup.
+
+### Python Protocol API
+- ``move_labware`` now requires api level 2.15; as a bonus feature to sweeten the deal, however, you can now `move_labware` to a special `OFF_DECK` location
+- The Mount type has an `EXTENSION` entry for the gripper now
+
+### ODD
+- You now get lovely little popups on the ODD when you send a protocol to an OT-3.
```
Comment (shlokamin) [inline api/release-notes-internal.md:25]: 😛 

```suggestion
- You now get lovely little popups on the ODD when you send a protocol to a Flex.
```

File: api/release-notes-internal.md [hunk 2]
```diff
@@ -12,10 +12,25 @@ Some things are known not to work, and are listed below. Specific compatibility
...
+- Many 96-channel behavior fixes, especially around tip pickup.
+
+### Python Protocol API
+- ``move_labware`` now requires api level 2.15; as a bonus feature to sweeten the deal, however, you can now `move_labware` to a special `OFF_DECK` location
+- The Mount type has an `EXTENSION` entry for the gripper now
+
+### ODD
+- You now get lovely little popups on the ODD when you send a protocol to an OT-3. 
+- Design passes on the following screens should improve little usability issues: protocols dashboard, connect to network, protocol details
+- The unboxing flow should handle USB connections better now
```
Comment (shlokamin) [inline api/release-notes-internal.md:27]: uh idt we touched the unboxing flow usb path

File: app-shell/build/release-notes-internal.md [hunk 3]
```diff
@@ -3,15 +3,18 @@ For more details about this release, please see the full [technical changelog][]
...
 
 This is still pretty early in the process, so some things are known not to work, and are listed below. Some things that may surprise you do work, and are also listed below. There may also be some littler things of note, and those are at the bottom.
 
+## Smaller Known Issues In This Release
+- Gripper calibration occasionally seems like it "skipped a step", going straight from front to rear calibration. If this happens, exit and rerun. This is because of a server error that isn't getting properly reported. We're working on both fixing the error and its reporting, but in the meantime exiting and rerunning the process should fix it. This is RQA-844.
+
 ## New Stuff In This Release
-- USB connectivity! If you connect your Flex via USB, it should show up.
-- LPC supports custom labware again
+- USB connectivity improvements: the app will display a nice big icon in the network settings page for a USB robot, USB robots will disappear properly when disconnected, things of this nature.
```
Comment (shlokamin) [inline app-shell/build/release-notes-internal.md:16]: same thing here re: usb

**Review by shlokamin — APPROVED** [review]
lgtm minus the app usb call outs cuz i dont see any file changes there

**Pull request merged**